src/locale/locales/fr/messages.po

```diff
@@ -2171,14 +2186,14 @@ msgstr "Cet avertissement n’est disponible que pour les messages contenant des
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:178
 msgid "This will hide this post from your feeds."
-msgstr ""
+msgstr "Cette opération permet de cacher cet article dans vos fils d'actualité."
```

Comment:
```suggestion
msgstr "Ceci cachera cet article dans vos fils d'actualité."
```

(Literally moving from `This operation enable hiding…` to `This will hide…`. I don't understand why this translation diverge that much from the original English text, I don't subscribe to that vision.)